Import Integrity-Policy WPT tests

yoavweiss opened the pull request:
#### 98733fd3822325456ee066f400c6465b5fc8dd91
<pre>
Import Integrity-Policy WPT tests
<a href="https://bugs.webkit.org/show_bug.cgi?id=293348">https://bugs.webkit.org/show_bug.cgi?id=293348</a>

Reviewed by Anne van Kesteren.

Upstream commit: <a href="https://github.com/web-platform-tests/wpt/commit/f5a18b7277984ee2392241f954a310fd9c6c6a90">https://github.com/web-platform-tests/wpt/commit/f5a18b7277984ee2392241f954a310fd9c6c6a90</a>

This PR adds the relevant Integrity-Policy tests.

* LayoutTests/imported/w3c/web-platform-tests/subresource-integrity/integrity-policy/parsing.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/subresource-integrity/integrity-policy/script.https.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/subresource-integrity/integrity-policy/w3c-import.log: Added.
* LayoutTests/tests-options.json: Mark tests as slow.
* LayoutTests/TestExpectations: Skip the tests for now.

Canonical link: <a href="https://commits.webkit.org/295261@main">https://commits.webkit.org/295261@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/1c732ef83f6f2db2ccddbdfd98941f5f6497070e

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/104450 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/24159 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/14577 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/109657 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/55121 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/24540 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/32704 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/79312 "Passed tests") | [⏳ 🧪 win-tests](https://ews-build.webkit.org/#/builders/Win-Tests-EWS "Waiting to run tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/107456 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/19082 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/94275 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/59638 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/133/builds/18875 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/12347 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/54484 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/88575 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/12405 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/112042 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/31611 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/23288 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/88499 "Passed tests") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/31975 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/90508 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/88017 "Passed tests") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/22443 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/32910 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/10689 "Passed tests") | [✅ 🛠 playstation](https://ews-build.webkit.org/#/builders/134/builds/26086 "Built successfully") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/31538 "Built successfully") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/31330 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/34669 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/32890 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

annevk reviewed (commented):
The TestExpectations entry should be skip. It's not worth spending resources on something that will timeout.

yoavweiss committed:
Import Integrity-Policy WPT tests
https://bugs.webkit.org/show_bug.cgi?id=293348

Reviewed by Anne van Kesteren.

Upstream commit: https://github.com/web-platform-tests/wpt/commit/f5a18b7277984ee2392241f954a310fd9c6c6a90

This PR adds the relevant Integrity-Policy tests.

* LayoutTests/imported/w3c/web-platform-tests/subresource-integrity/integrity-policy/parsing.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/subresource-integrity/integrity-policy/script.https.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/subresource-integrity/integrity-policy/w3c-import.log: Added.
* LayoutTests/tests-options.json: Mark tests as slow.
* LayoutTests/TestExpectations: Skip the tests for now.

Canonical link: https://commits.webkit.org/295261@main
